submit-web/src/components/Shared/Forms/common.tsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+
+type FormProps = {
+  onSubmit: (event: React.FormEvent<HTMLFormElement>) => void;
+  children: React.ReactNode;
+} & React.FormHTMLAttributes<HTMLFormElement>;
+
+export default function Form({ onSubmit, children, ...rest }: FormProps) {
+  const handleKeyDown = (event: React.KeyboardEvent) => {
+    if (
+      event.key === "Enter" &&
+      (event.target as HTMLElement).tagName === "INPUT"
+    ) {
+      event.preventDefault();
+    }
+  };
+
+  return (
+    <form onSubmit={onSubmit} onKeyDown={handleKeyDown} {...rest}>
```

Comment:
do u need to explicitly pass onSubmit? can't it be part of rest

Reply:
good catch